Sources/Navigator/EPUB/EPUBSpreadView.swift

```diff
@@ -382,6 +389,21 @@ class EPUBSpreadView: UIView, Loggable, PageView {
         }
     }
 
+    private func keyEventHandler(_ event: Any) {
```

Comment:
To follow the same convention as the other event handlers, you can rename this one like this:
```suggestion
    private func didPressKey(_ event: Any) {
```